crawl-ref/source/rltiles/dc-spells.txt

```diff
@@ -262,6 +263,7 @@ summon_ufetubus SUMMON_UFETUBUS
 summon_tzitzimitl SUMMON_TZITZIMITL
 summon_undead SUMMON_UNDEAD
 summon_vermin SUMMON_VERMIN
+teleport TELEPORT
```

Comment:
This line should be removed now that the wand of warping has its own spell tile, and doesn't use the teleport tile anymore.